RNTester/NativeModuleExample/ScreenshotMacOS.h

```diff
@@ -5,17 +5,29 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-#import <React/RCTViewManager.h>
-#import <ReactCommon/RCTTurboModuleManager.h>
+#import <NativeModules.h>
 
-@interface ScreenshotManagerTurboModuleManagerDelegate : NSObject<RCTTurboModuleManagerDelegate>
-- (std::shared_ptr<facebook::react::TurboModule>)
-  getTurboModule:(const std::string &)name
-  jsInvoker:(std::shared_ptr<facebook::react::CallInvoker>)jsInvoker;
+REACT_STRUCT(ScreenshotArguments)
+struct ScreenshotArguments
+{
+};
 
-- (std::shared_ptr<facebook::react::TurboModule>)
-  getTurboModule:(const std::string &)name
-  instance:(id<RCTTurboModule>)instance
-  jsInvoker:(std::shared_ptr<facebook::react::CallInvoker>)jsInvoker;
+REACT_MODULE(ScreenshotManagerCxx, L"ScreenshotManager")
+struct ScreenshotManagerCxx
+{
+  REACT_INIT(Initialize)
+  void Initialize(const winrt::Microsoft::ReactNative::ReactContext& reactContext) noexcept
```

Comment:
What is this `winrt` type we're using?

Reply:
@ZihanChen-MSFT added this starting with #595 (hoping we can change the namespace in the near future!)

Reply:
Thanks for the backstory! Seeing anything `winrt` specific on Apple platforms makes me cringe. Could we open a bug on this to change it just so we're tracking that namespace renaming work?

Reply:
Captured this in #732 and assigned to @ZihanChen-MSFT 😀

Reply:
> Could we open a bug on this to change it just so we're tracking that namespace renaming work?

Related CPP files are mirrored from react-native-windows, in order to remove the winrt namespace, it requires works on that repo to be done first, I don't expect this could happen recently